src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import useStyles from './styles';
+import Radio from '@material-ui/core/Radio';
+import { RadioProps } from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+const RadioButton: React.FC<RadioProps> = (props) => {
```

Comment:
- This should extend Mui Radio button
- One radio button is of no use...and they generally come in groups...so if one of the group is selected the others are turned off (this behaviour is customizable with multiple selections)
Refer to Mui docs for more details

Reply:
@arkajyotiMukherjee Can you please explain the first point, as I've tried with interface, but later @S-ayanide told me to remove it. I don't see nay other method apart from this one. If you know any other way, let me know please.

Reply:
I hope the above answers your question

Reply:
@S-ayanide Currently we don't have interface. Have extended the RadioProps. I'd like to know if this variant is right or not?

Reply:
Can you please take a look at this example! https://material-ui.com/components/radio-buttons/#radiogroup

We want to imitate the checked functionality which can be done with or without RadioGroup wrapper